src/hooks/useSortColumnManager/useSortColumnManager.js

```diff
@@ -17,7 +17,7 @@ const useSortColumnManager = (options) => {
 
     let newOrder;
 
-    if (options?.sortableColumns && !options?.sortableColumns.includes(name)) return;
+    if (options?.sortableColumns && !Object.values(options.sortableColumns).includes(name)) return;
```

Comment:
In tests we're passing `sortableColumns` as an array, but `sortableSearchResultListColumns` is an object.
I'd prefer this to be an array so we don't have to use `Object.values`.
And after changing it to array we also should update how it's used in `AuthoritiesSearch`